app/wama.core/ViewModel/User/UserAccountViewModel.cs

```diff
@@ -6,7 +6,7 @@ namespace WAMA.Core.ViewModel.User
     public abstract class UserAccountViewModel
     {
         public virtual UserAccountType AccountType { get; }
-
+        [Required]
```

Comment:
That's not what I se.
When I leave out any of the required fields, the browser stops me.

If I make it past the browser, I will also get stopped.
Because model binding will fail, since I marked those properties with the [Required] attribute.